app/core/steps/EligibilityValidationStep.js

```diff
@@ -6,15 +6,23 @@ const eligibilityCookie = new EligibilityCookie();
 
 class EligibilityValidationStep extends ValidationStep {
 
-    getContextData(req, res, pageUrl, fieldKey) {
-        const ctx = super.getContextData(req);
+    setFeatureTogglesOnCtx(ctx, featureToggles = {}) {
+        Object.keys(featureToggles).forEach((toggle) => {
+            ctx[toggle] = featureToggles[toggle];
+        });
+        return ctx;
+    }
+
+    getContextData(req, res, pageUrl, fieldKey, featureToggles) {
+        let ctx = super.getContextData(req);
 
         if (req.method === 'GET') {
             const answerValue = eligibilityCookie.getAnswer(req, pageUrl, fieldKey);
             if (answerValue) {
                 ctx[fieldKey] = answerValue;
             }
         } else {
+            ctx = this.setFeatureTogglesOnCtx(ctx, featureToggles);
```

Comment:
Can this line be added to one of the unit tests.

So that when `featureToggles` with a toggle value is passed into `getContextData()` we can see that it gets set onto the context. It doesn't need to test multiple toggle values, just one will be fine.